modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/GoClientCodegen.java

```diff
@@ -434,6 +434,15 @@ public String toOperationId(String operationId) {
         return objs;
     }
 
+    @Override
+    public void postProcessModelProperty(CodegenModel model, CodegenProperty property) {
```

Comment:
This shouldn't be done as an override. Look at how the CLI SDK assigns the int type mapping and use the same approach here.

Reply:
The thinking here was that there is no reason for changing other int32 and int64 variables to ints unnecessarily. That might lead other problems in future such as overflows etc

Reply:
There shouldn't be a possibility for overflows. Per https://tour.golang.org/basics/11:

> The int, uint, and uintptr types are usually 32 bits wide on 32-bit systems and 64 bits wide on 64-bit systems. When you need an integer value you should use int unless you have a specific reason to use a sized or unsigned integer type.

Per [the swagger 2.0 data types](https://swagger.io/specification/v2/), the `integer` type in swagger describes a signed 32 bit integer. Go's definition of `int` should be compatible with all swagger integer properties. Properties in swagger that require a 64 bit integer must be of swagger type `long`.

Reply:
Yeah, that's fair enough so. Looks like the override is a bit of an overkill then.